AllJoyn/Platform/Bridge.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.24720.0
+# Visual Studio 15
```

Comment:
We probably shouldn't force everyone to upgrade to a prerelease version of VS. Or is it needed for some reason?